search-app/src/cmr/search/site/pages.clj

```diff
@@ -0,0 +1,36 @@
+(ns cmr.search.site.pages
+  "The functions of this namespace are specifically responsible for returning
+  ready-to-serve pages."
+  (:require
+    [cmr.search.site.data :as data]
+    [selmer.parser :as selmer]))
+
+(defn render-template-ok
+  "A utility function for preparing template pages."
+  [template data]
+  {:status 200
+   :body (selmer/render-file template data)})
+
+(defn home
+  "Prepar the home page template."
+  [request]
+  (render-template-ok
+    "templates/index.html"
+    (data/get-index request)))
+
+(defn landing-links
+  "Prepare the page that links to all landing page links.
+
+  For now, this is just a page with a single link (the EOSDIS collections
+  landing pages)."
+  [request]
+  (render-template-ok
```

Comment:
Yeah, the significance is that the function explicitly sets a status of 200 -- if we had a bunch of custom 404 templates, I'd write a `render-template-not-found` that wrapped up the status and body in a similar manner.